main.js

```diff
@@ -0,0 +1,54 @@
+ const electron = require("electron");
+// Module to control application life.
+ const {app} = electron;
```

Comment:
Yeah, I don't see any value in restructuring the object if we're just assigning it to a single variable